pkg/cmd/attestation/verify/verify_integration_test.go

```diff
@@ -138,6 +156,16 @@ func TestVerifyIntegrationReusableWorkflow(t *testing.T) {
 		require.NoError(t, err)
 	})
 
+	t.Run("with repo and valid reusable signer workflow", func(t *testing.T) {
+		opts := baseOpts
+		opts.Owner = "malancas"
+		opts.Repo = "malancas/attest-demo"
+		opts.SignerWorkflow = "https://github.com/github/artifact-attestations-workflows/.github/workflows/attest.yml@09b495c3f12c7881b3cc17209a327792065c1a1d"
```

Comment:
I'm open to feedback here, but I think we want the `https://github.com/` to come from the CLI (in a Proxima-compatible way), and not be required to be supplied by the user.

Reply:
That makes sense to me. The cert-identity flag currently assumes the user has provided the https://github.com. Do you want to update that flag too to no longer expect that prefix?

Reply:
While I think we can discuss this more in the Proxima cli ADR, many commands in the cli accepts `[HOST/]OWNER/REPO` and I think that could be a nice one to follow, as that can be used to untangle the possible confusion if multiple hosts are known to the cli. 

In short, what I'm saying is that we should allow the user to specify `https://github.com`, and if not use the first configured host in the config. If `GH_HOST` env var is specified, that should be used instead. That's how I understood the rest of the cli works?